README.md

```diff
@@ -82,7 +82,7 @@ Diagnostic information includes:
 * Collectable metrics (for collector plugins only)
 * How long it took to run each of these diagnostics
 
-Currently Snap Plugin Diagnostics is only available for collector plugins.
+Currently Snap Plugin Diagnostics is only available for collector and streaming collector plugins.
```

Comment:
I thin plugin diagnostics should't be available for streaming collectors.

Reply:
You're right. Diagnostics are not enabled for streaming collectors at this time. I will fix this.